test/test_suites.py

```diff
@@ -112,91 +104,114 @@ def create_end_to_end_test_suites(driver, nameSalt, schemaRegistryAddress, testS
             test_instance=TestAvrosrAvrosr(driver, nameSalt), clean=True, run_in_confluent=True, run_in_apache=False
         )),
         ("TestNativeStringAvrosr", EndToEndTestSuite(
-            test_instance=TestNativeStringAvrosr(driver, nameSalt), clean=True, run_in_confluent=True, run_in_apache=False
+            test_instance=TestNativeStringAvrosr(driver, nameSalt), clean=True, run_in_confluent=True,
```

Comment:
nit: why are all these 'run_in_apache' lines pushed down a line? i don't see any other differences, is there a formatting change?

Reply:
all the formatting is done automatically so I have no control